src/ImageSharp/Common/Helpers/UnitConverter.cs

```diff
@@ -100,12 +100,11 @@ public static PixelResolutionUnit ExifProfileToResolutionUnit(ExifProfile profil
         /// <summary>
         /// Sets the exif profile resolution values.
         /// </summary>
-        /// <param name="exifProfile">The exif profile.</param>
         /// <param name="unit">The resolution unit.</param>
         /// <param name="horizontal">The horizontal resolution value.</param>
         /// <param name="vertical">The vertical resolution value.</param>
         [MethodImpl(InliningOptions.ShortMethod)]
-        public static void SetResolutionValues(ExifProfile exifProfile, PixelResolutionUnit unit, double horizontal, double vertical)
+        public static (ushort, double?, double?) AdjustToExif(PixelResolutionUnit unit, double horizontal, double vertical)
```

Comment:
I don't really like the signature of this method (return type), I'll be glad to suggestions

Reply:
No that seems off both in naming of the method and the return type. I'd return an explicitly declared readonly struct.

Reply:
Done! Please review